development/kyma-ci-force-bot/kyma-ci-force-bot/prod-values.yaml

```
@@ -16,7 +16,7 @@ eventType: "issuesevent.closed.v1"
 container:
   name: "ciforcebot"
   # change to match value from makefile
-  image: eu.gcr.io/kyma-project/test-infra/pr/ci-force-bot:PR-4216
+  image: eu.gcr.io/kyma-project/test-infra/pr/ci-force-bot:PR-4473
```

Shouldn't this image also use non-pr registry?
```suggestion
  image: eu.gcr.io/kyma-project/test-infra/ci-force-bot:PR-4473
```

---

Corrected to use image from main. Updated tag to the latest one.